src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -33,4 +38,18 @@
                       PublishFlatContainer="$(PublishFlatContainer)"
                       Overwrite="$(Overwrite)" />
     </Target>
-</Project>
+
+  <!-- This is the target used to configure feeds for restore. It can be used to specify the sources from which you would restore your packages from.
+       Parameters to specify:
+       EnableFeeds - ItemGroup which contains sources which are allowed
+      ConfigureInputDependsOn can be customized at build time on the commandline to determine the pre steps required
+  -->
+
+  <Target Name="ConfigureInputFeeds" DependsOnTargets="@(ConfigureInputDependsOn)" >
```

Comment:
Do we still have a needed for this target?

Reply:
Unclear if we will at this point, all repos that we did update have a checked in Nuget.config.

Reply:
OK we can keep it for now and wait to see if folks end up needing it.